float64 posicion_x
float64 posicion_y
float32[] obj_x
float32[] obj_y
uint32 width
uint32 height
float32 resolution
-----------------


